sway/commands/output/transform.c

```diff
@@ -32,8 +33,22 @@ struct cmd_results *output_cmd_transform(int argc, char **argv) {
 		return cmd_results_new(CMD_INVALID, "output",
 			"Invalid output transform.");
 	}
-
 	config->handler_context.leftovers.argc = argc - 1;
 	config->handler_context.leftovers.argv = argv + 1;
+
+	if (argc == 2) {
+		if (strcmp(argv[1], "clockwise") == 0) {
+			output->transform_type = TRANSFORM_RELATIVE_CLOCKWISE;
+		} else if (strcmp(argv[1], "anticlockwise") == 0) {
+			output->transform_type = TRANSFORM_RELATIVE_ANTICLOCKWISE;
+		} else {
+			return cmd_results_new(CMD_INVALID, "output",
+				"Invalid output transform type.");
```

Comment:
This doesn't work if you chain output commands like so:

```
output XXX transform 90 scale 2
```

Reply:
Oh right, will tidy the detection up.